CONTRIBUTING.md

```diff
@@ -1,69 +1,138 @@
-# Fork Status
+# Contribution guidelines
 
-## Pull Requests
+First off, thank you for considering contributing to ratatui.
```

Comment:
Capitalize Ratatui or surround it in single backticks, whichever is most consistent with the rest of the documentation/other written materials.